airbyte-cdk/python/airbyte_cdk/sources/streams/http/exceptions.py

```diff
@@ -32,6 +32,12 @@ class BaseBackoffException(requests.exceptions.HTTPError):
     pass
 
 
+class RequestBodyException(requests.exceptions.HTTPError):
```

Comment:
From the code bellow, this appears to be the local exception and it's not related to any of `requests.exceptions.HTTPError`, maybe it's better to inherit from just `Exception` as we're not going to face with actual `requests.exceptions.HTTPError` error codes,  since we don't send/receive any request yet.

Reply:
Agree